lib/server.js

```diff
@@ -4,6 +4,29 @@ const Hapi = require('hapi');
 const registrationMan = require('./registerPlugins');
 
 /**
+ * If we're throwing errors, let's have them say a little more than just 500
+ * @method prettyPrintErrors
+ * @param  {Hapi.Request}    request Hapi Request object
+ * @param  {Hapi.Reply}      reply   Hapi Reply object
+ */
+function prettyPrintErrors(request, reply) {
+    if (request.response.isBoom) {
+        const err = request.response;
+        const errName = err.output.payload.error;
+        const errMessage = err.message;
```

Comment:
What is the difference between `err.message` and `err.output.payload.message`? Can this method just `return reply(err.output.payload).code(statusCode)`?